common/templates/layouts/base.njk

```diff
@@ -153,9 +153,9 @@
   
   {% if FEATURE_FLAGS.WHATS_NEW_BANNER and REQUEST_PATH === '/' %}
     {{ appWhatsNewBanner({
-      title: whatsNewContent.title,
-      body: whatsNewContent.bannerText,
-      date: whatsNewContent.date
+      title: whatsNewContent.bannerContent.title,
+      body: whatsNewContent.bannerContent.body,
+      date: whatsNewContent.bannerContent.date
```

Comment:
```suggestion
      whatsNewContent.bannerContent
```

I think we can do `appWhatsNewBanner(whatsNewContent.bannerContent)` now without the need to pass the variables individually.